artifactor/plugins/ostriz.py

```diff
@@ -7,7 +7,7 @@
 
 def overall_test_status(statuses):
     # Handle some logic for when to count certain tests as which state
-    for when, status in statuses.items():
+    for when, status in list(statuses.items()):
```

Comment:
that kind of false positive is to be avoided ^^

Reply:
This is a result of 2to3's `dict` checker, unfortunately as far as I can tell this syntax change can't be separated from that checker.

Reply:
Went through and removed them in a54e30bf3